datafusion-cli/src/main.rs

```diff
@@ -72,6 +69,14 @@ pub async fn main() {
                 .validator(is_valid_format)
                 .takes_value(true),
         )
+        .arg(
+            Arg::with_name("timing")
```

Comment:
What do you think about making this option `quiet` or something like that and having it turn off all output (other than query results) and not just the timing information?

Reply:
a good point - let me change to that